### PR TITLE
[PS-683] Fix missed defaults and sentence cases

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

1. I missed some changes and some typos in the initial PR, this fixes those.
2. After discussions with @danielleflinn the "Ask for biometrics on launch" checkbox is now hidden if biometrics are disabled (see: https://bitwarden.atlassian.net/browse/PS-684)
3. I refactored biometric checkbox to use `ngModel`. Also added a note about a bug that's is affecting the checkbox (the bug is not introduced in this PR, it's been there since before). The bug affects `[checked]` as well as `[ngModel]`.

## Code changes

- **apps/browser/src/popup/settings/settings.component.html:** Hide auto prompt checkbox if biometrics disabled.
- **apps/browser/src/popup/settings/settings.component.ts :** Remove superfluous default. State service already contains default values.
- **apps/desktop/src/app/accounts/settings.component.html :** Hide auto prompt checkbox if biometrics disabled. Use `ngModel` for biometric checkbox.
- **apps/desktop/src/app/accounts/settings.component.ts :** Refactor `updateBiometric` to make it easier to read. Remove if-guard in `updateAutoPromptBiometrics` which is no longer needed because we hide the checkbox.
- **apps/desktop/src/locales/en/messages.json :** Fix typo and translation I missed intially.

## Screenshots

image
image

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: apps/desktop/src/app/accounts/settings.component.ts [hunk 1]
```diff
@@ -265,32 +265,38 @@ export class SettingsComponent implements OnInit {
     }
   }
 
-  async updateBiometric() {
-    const current = this.biometric;
-    if (this.biometric) {
+  async updateBiometric(newValue: boolean) {
+    // NOTE: A bug in angular causes [ngModel] to not reflect the backing field value
+    // causing the checkbox to remain checked even if authentication fails.
+    // See: https://github.com/angular/angular/issues/13063
```
Comment (MGibson1) [inline apps/desktop/src/app/accounts/settings.component.ts:271]: Can you describe what to change to get rid of this workaround once that issue is closed?

File: apps/desktop/src/app/accounts/settings.component.ts [hunk 2]
```diff
@@ -265,32 +265,38 @@ export class SettingsComponent implements OnInit {
...
+      await this.stateService.setBiometricLocked(false);
+      await this.cryptoService.toggleKey();
+      return;
     }
-    if (this.biometric === current) {
+
+    if (!this.supportsBiometric) {
+      this.biometric = false;
       return;
     }
```
Comment (MGibson1) [inline apps/desktop/src/app/accounts/settings.component.ts:284]: Maybe merge these two ifs with an `||` ?

**Review by MGibson1 — CHANGES_REQUESTED** [review]

File: apps/desktop/src/app/accounts/settings.component.ts (hunk 1 above)
Comment (coroiu) [reply apps/desktop/src/app/accounts/settings.component.ts:271]: Once the issue is closed it should resolve itself, I'll add a comment mentioning that!

File: apps/browser/src/popup/settings/settings.component.ts [hunk 3]
```diff
@@ -114,9 +114,7 @@ export class SettingsComponent implements OnInit {
 
     this.supportsBiometric = await this.platformUtilsService.supportsBiometric();
     this.biometric = await this.vaultTimeoutService.isBiometricLockSet();
-    const disableAutoBiometricsPrompt =
-      (await this.stateService.getDisableAutoBiometricsPrompt()) ?? true;
-    this.enableAutoBiometricsPrompt = !disableAutoBiometricsPrompt;
+    this.enableAutoBiometricsPrompt = !(await this.stateService.getDisableAutoBiometricsPrompt());
```
[the remark this answers is not in the document]
Comment (coroiu) [reply apps/browser/src/popup/settings/settings.component.ts:117]: Yeah but `getDisableAutoBiometricsPrompt` already contains a null coalescing operator, so this operator should be unnecessary, right? See: https://github.com/bitwarden/clients/blob/PS-683-fix-defaults-and-sentence-cases/libs/common/src/services/state.service.ts#L865

File: apps/desktop/src/app/accounts/settings.component.ts [hunk 4]
```diff
@@ -265,32 +265,38 @@ export class SettingsComponent implements OnInit {
...
+    await this.stateService.setBiometricUnlock(true);
     await this.stateService.setBiometricLocked(false);
     await this.cryptoService.toggleKey();
   }
 
   async updateAutoPromptBiometrics() {
-    if (!this.biometric) {
-      this.autoPromptBiometrics = false;
-    }
-
```
[the remark this answers is not in the document]
Comment (coroiu) [reply apps/desktop/src/app/accounts/settings.component.ts:293]: I can re-add it, but will this case ever actually happen? This method is only called through the checkbox, and the checkbox is completely removed from the DOM if `this.biometric` is `false`

File: apps/browser/src/popup/settings/settings.component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (MGibson1) [reply apps/browser/src/popup/settings/settings.component.ts:117]: ah, most things that come out of that class can be null. yeah, you don't need it

File: apps/desktop/src/app/accounts/settings.component.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (MGibson1) [reply apps/desktop/src/app/accounts/settings.component.ts:293]: Removing an element from the dom doesn't remove it from the backing component. It'll maintain state even without a visualization.

**Pull request merged**